test/proxy.spec.ts

```diff
@@ -0,0 +1,59 @@
+import { Tests } from './lib/tests';
+import { HttpCall } from './lib/types';
+
+const tests = new Tests('proxy');
+
+const getAnswerCall: HttpCall = {
+  description: 'Call GET answer',
+  path: '/answer',
+  method: 'GET',
+  testedProperties: {
+    body: '42',
+    status: 200
+  }
+};
+
+const environmentLogsItemSelector = '.environment-logs-column:nth-child(1) .menu-list .nav-item';
+
+describe('Proxy', () => {
+  tests.runHooks();
+
+  it('Start environments', async () => {
+    await tests.helpers.startEnvironment();
+    await tests.helpers.selectEnvironment(2);
+    await tests.helpers.startEnvironment();
+  });
+
+  it ('Call /anwser', () => {
```

Comment:
please use async / await to avoid race conditions

Reply:
Done.